src/amadeus/namespaces/booking/flight_orders.js

```diff
@@ -0,0 +1,494 @@
+                    "quantity": 1
+                  }
+                }
+              ]
+            }
+          ]
+        }
+      ],
+      "travelers": [
+        {
+          "id": "1",
+          "dateOfBirth": "1982-01-16",
+          "name": {
+            "firstName": "JORGE",
+            "lastName": "GONZALES"
+          },
+          "gender": "MALE",
+          "contact": {
+            "emailAddress": "jorge.gonzales833@telefonica.es",
+            "phones": [
+              {
+                "deviceType": "MOBILE",
+                "countryCallingCode": "34",
+                "number": "480080076"
+              }
+            ]
+          },
+          "documents": [
+            {
+              "documentType": "PASSPORT",
+              "birthPlace": "Madrid",
+              "issuanceLocation": "Madrid",
+              "issuanceDate": "2015-04-14",
+              "number": "00000000",
+              "expiryDate": "2025-04-14",
+              "issuanceCountry": "ES",
+              "validityCountry": "ES",
+              "nationality": "ES",
+              "holder": true
+            }
+          ]
+        },
+        {
+          "id": "2",
+          "dateOfBirth": "2012-10-11",
+          "gender": "FEMALE",
+          "contact": {
+            "emailAddress": "jorge.gonzales833@telefonica.es",
+            "phones": [
+              {
+                "deviceType": "MOBILE",
+                "countryCallingCode": "34",
+                "number": "480080076"
+              }
+            ]
+          },
+          "name": {
+            "firstName": "ADRIANA",
+            "lastName": "GONZALES"
+          }
+        }
+      ],
+      "remarks": {
+        "general": [
+          {
+            "subType": "GENERAL_MISCELLANEOUS",
+            "text": "ONLINE BOOKING FROM INCREIBLE VIAJES"
+          }
+        ]
+      },
+      "ticketingAgreement": {
+        "option": "DELAY_TO_CANCEL",
+        "delay": "6D"
+      },
+      "contacts": [
+        {
+          "addresseeName": {
+            "firstName": "PABLO",
+            "lastName": "RODRIGUEZ"
+          },
+          "companyName": "INCREIBLE VIAJES",
+          "purpose": "STANDARD",
+          "phones": [
+            {
+              "deviceType": "LANDLINE",
+              "countryCallingCode": "34",
+              "number": "480080071"
+            },
+            {
+              "deviceType": "MOBILE",
+              "countryCallingCode": "33",
+              "number": "480080072"
+            }
+          ],
+          "emailAddress": "support@increibleviajes.es",
+          "address": {
+            "lines": [
+              "Calle Prado, 16"
+            ],
+            "postalCode": "28014",
+            "cityName": "Madrid",
+            "countryCode": "ES"
+          }
+        }
+      ]
+    }
+  }));
+   * ```
+   */
+  post(params = {}) {
+    return this.client.post('/v1/booking/flight-orders', params);
```

Comment:
Also, to help the user call the API easier, we build the POST body. Then the user has just to give as parameter the `flight` and `traveler` that are mandatory. Have a look at the implementation [here](https://github.com/amadeus4dev/amadeus-python/blob/master/amadeus/booking/_flight_orders.py).

Reply:
As discussed, leaving a jsonBody as param for this method.